src/NServiceBus.Core/Scheduling/Schedule.cs

```diff
@@ -9,7 +9,7 @@ namespace NServiceBus
     /// <summary>
     /// Extends the context with scheduling capabilities.
     /// </summary>
-    public static class ScheduleBusExtensions
+    public static class ScheduleExtensions
```

Comment:
Just wanted to point out that with the using static feature of C# 6, there are potentially reasons to directly reference an extensions-only class.

As @timbussmann pointed out, since this lives directly in the `NServiceBus` namespace it's not as likely to be referenced separately.